tests/testthat/test-use_r_workflows.R

```diff
@@ -73,6 +73,26 @@ test_that("use_doc_and_style_r() works with manual trigger", {
   expect_true(length(grep("workflow_dispatch:", test, fixed = TRUE)) > 0)
 })
 
+test_that("use_doc_and_style_r() works with pat option", {
+  use_doc_and_style_r(
+    workflow_name = "doc_style_pat.yml",
+    use_rm_dollar_sign = FALSE, how_to_commit = "pull_request", 
+    use_pat = TRUE, pat_name = "MYPAT"
+  )
+  expect_true(file.exists(".github/workflows/doc_style_pat.yml"))
+  test <- readLines(".github/workflows/doc_style_pat.yml")
```

Comment:
comment [non-blocking]: maybe testing output of the R functions with [snapshot tests](https://testthat.r-lib.org/articles/snapshotting.html)

Reply:
great point, I just learned about them from the ones (I assume) you wrote for FIMS!